tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/lantern/test.js

```diff
@@ -26,7 +26,7 @@ nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", functio
         type: "Model",
         // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Lantern
         modelURL: assetsRootPath + 'models/gltf_models/glb/Lantern.glb',
-        position: Vec3.sum(position, {x: 0.0, y: 0.2, z: -1.6 }),
+        position: Vec3.sum(position, {x: 0.0, y: 0.8, z: -2.75 }),

```

Comment:
Why'd you make this one farther from the camera?

Reply:
fixing the bounding boxes also fixed the positioning of a bunch of the models. instead of scaling down the model further to fit it entirely in frame, I pushed it back (I'll double check though; it's possible that it doesn't need to be pushed back as much to fit in frame)